nima/tests/integration/http2/server/src/test/resources/logging-test.properties

```diff
@@ -15,7 +15,10 @@
 #
 handlers=java.util.logging.ConsoleHandler
 java.util.logging.ConsoleHandler.level=FINEST
+java.util.logging.ConsoleHandler.formatter=java.util.logging.SimpleFormatter
```

Comment:
Please do not add commented out lines in

Reply:
ok